Dynamic-Programming/ContinuousSubarraySum/README.md

```diff
@@ -0,0 +1,112 @@
+## Continous Subarray Sum
+
+<p>
+<img align="center" width="800px" alt="Question Screenshot" src="./../../Images/ContinuousSubarraySum/question.png">
+</p>
+
+---
+
+###  Motivation
+We are given an array of Integers `nums` and `K`. we have to find a continuous subset of the array of `length>=2` whose sum is of the form `N*K` where `N` is any integer. 
+
+
+### Solution-1 : Naive Way
+
+The naive solution involves traversing all the subset of `nums` whose `length>=2` and checking for `nums[i:j]%K==0` where `i<j` and `nums[i:j]= nums[i]+nums[i+1]+...+nums[j-1]+nums[j]`. While traversing the subsets we explicitily need handle the case `K=0` because `nums[i:j]%K` will throw `DivideByZeroException`.
```

Comment:
While traversing the subsets we ~explicitily~ **explicitly to** need handle the case